test/cmd/env.sh

```diff
@@ -21,5 +21,27 @@ os::cmd::expect_success_and_text 'oc set env dc/node --env PASS=x,y=z' 'no longe
 # warning is not printed for variables passed as positional arguments
 os::cmd::expect_success_and_not_text 'oc set env dc/node PASS=x,y=z' 'no longer accepts comma-separated list'
 
+# create a build-config object with the JenkinsPipeline strategy
+os::cmd::expect_success 'oc create -f test/templates/testdata/jenkins-ephemeral.json'
```

Comment:
why can't you use this out of examples/jenkins ?

Reply:
Thanks, switched to using the template in examples/jenkins